File: HELLO_JAVA/src/day02/myholl.java
package day02;

import java.util.Scanner;

public class myholl {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		while(true) {
		System.out.print("홀짝을 선택해주세요.");
		String user = sc.next();
		int rand = (int) Math.random()*100;
		String com ="";
		if(rand>50) {
			com = "홀";
		}else {
			com = "짝";
		}
		
		System.out.println(com);
		
		if(com.equals(user)) {
			System.out.println("이겼습니다.");
		}else {
			System.out.println("졌습니다.");
		}
		}
	}
	
}

File: HELLO_PYTHON/day02/my_holl.py
from random import random

a = input("홀짝을 선택해 주세요.")
rand = random()
if rand<0.5 :
    com = "홀"
else :
    com ="짝"


print("컴퓨터:",com)
if a==com :
    print("이겼습니다.")
else :
    print("졌습니다.")
    
